src/style.css

```diff
@@ -2,2 +1,72 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.site-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 70px;
+  background-color: #fff;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.logo img {
+  height: 50px;
+  width: auto;
+  display: block;
+  margin-left: 20px;
+}
+
+.main-nav {
+  display: block;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
   margin: 0;
+  padding: 0;
+}
+
+.nav-item {
+  text-decoration: none;
+  color: black;
+  margin-right: 20px;
+  height: 70px;
+  line-height: 70px;
+  display: inline-block;
+  position: relative;
+  text-align: center;
+}
```

Comment:
The task requires all navigation items to be displayed in uppercase using CSS. Consider adding `text-transform: uppercase;` to the `.nav-item` class.